taxi/tests/test_models.py

```diff
@@ -0,0 +1,64 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Car
+
+
+class ModelTests(TestCase):
+    def test_manufacturer_str(self) -> None:
+        manufacturer = Manufacturer.objects.create(
+            name="test_name",
+            country="test_country"
+        )
+        self.assertEquals(
+            str(manufacturer),
+            f"{manufacturer.name} {manufacturer.country}"
+        )
+
+    def test_driver_str(self) -> None:
+        driver = get_user_model().objects.create(
+            username="test",
+            password="test123",
+            first_name="test_first",
+            last_name="test_last"
+        )
+        self.assertEquals(
+            str(driver),
+            f"{driver.username} ({driver.first_name} {driver.last_name})"
+        )
+
+    def test_car_str(self) -> None:
+        manufacturer = Manufacturer.objects.create(
+            name="test_name",
+            country="test_country"
+        )
+
+        driver = get_user_model().objects.create(
+            username="test",
+            password="test123",
+            first_name="test_first",
+            last_name="test_last"
+        )
+
+        car = Car.objects.create(
+            model="test_model",
+            manufacturer=manufacturer,
+        )
+        car.drivers.set([driver])
+
+        self.assertEquals(str(car), car.model)
+
+    def test_create_driver_with_license_number(self) -> None:
+        username = "test"
+        password = "test123"
+        license_number = "TES77777"
+
+        driver = get_user_model().objects.create_user(
+            username=username,
+            password=password,
+            license_number=license_number
+        )
+
+        self.assertEquals(driver.username, username)
+        self.assertEquals(driver.license_number, license_number)
```

Comment:
Consider using `self.assertEqual` instead of `self.assertEquals` for consistency with Django's testing framework. `assertEqual` is the preferred method name.